lang/python/python-lru-dict/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-lru-dict
+PKG_VERSION:=1.3.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=lru-dict
+PKG_HASH:=54fd1966d6bd1fcde781596cb86068214edeebff1db13a2cea11079e3fd07b6b
+
+PKG_MAINTAINER:=Timothy Ace <openwrt@timothyace.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=python-poetry-core/host python-cython/host python-wheel/host
+
```

Comment:
```suggestion
```

Upstream is [using setuptools](https://github.com/amitdev/lru-dict/blob/v1.3.0/pyproject.toml#L37) so this isn't necessary.

Reply:
This change was made. Tested recompiling = success.